public/app/features/dashboard/components/PanelEditor/getVizualizationOptions.tsx

```diff
@@ -109,7 +124,7 @@ export function fillOptionsPaneItems(
   optionEditors: PanelOptionsEditorItem[],
   getOptionsPaneCategory: categoryGetter,
   onValueChanged: (path: string, value: any) => void,
-  context: StandardEditorContext<any>
+  context: StandardEditorContext<any> & { builder: PanelOptionsEditorBuilder<any> }
```

Comment:
Here comes the trickiness - think this will require some types refactoring as the FieldOverrideContext is using StndardEditorContext and we cannot add the builder to it when the field overrides are applied. Maybe builder should be optional in the StandardEditorContext? But this will make working with it not very pelaseant. I guess whatever is passed down to the editor props should be non-optionsl